coriolis/db/api.py

```diff
@@ -497,22 +515,22 @@ def get_replica_migrations(context, replica_id):
 
 @enginefacade.reader
 def get_migrations(context, include_tasks=False,
-                   include_info=False, to_dict=True):
```

Comment:
Can you please make triple-sure that the `to_dict` kwargs' default changing doesn't cause issues with any other API call (most notably, the ones performed by the UI?

Reply:
This method is only used when listing migrations, and I have `to_dict` explicitly set to True there. No UI issues.